# exception.py

class CustomException(Exception):
    """Custom exception class for handling custom exceptions"""
    pass
